frontend/server/k8s-helper.ts

```diff
@@ -95,13 +96,14 @@ function getNameOfViewerResource(logdir: string): string {
  */
 export async function newTensorboardInstance(
   logdir: string,
+  tfversion: string,
   podTemplateSpec: Object = defaultPodTemplateSpec,
 ): Promise<void> {
   if (!k8sV1CustomObjectClient) {
     throw new Error('Cannot access kubernetes Custom Object API');
   }
   const currentPod = await getTensorboardInstance(logdir);
-  if (currentPod) {
+  if (currentPod.podAddress) {
```

Comment:
If `currentPod.tfVersion` doesn't match tfVersion passed in this time, better give an error `There's already an existing tensorboard instance with a different version...`